tutorials/market/Market_Intelligence_Part2.ipynb

```diff
@@ -250,7 +250,7 @@
    "cell_type": "markdown",
```

Comment:
<p>I like this dependency parser description now, easier to understand</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>